### DISCUSSION TITLE
Disable hover effect for widget

### DISCUSSION THREAD

**Discussion by lowfrank** [emilk/egui#2116] [Q&A] [answered]
I can't find a way to disable the effect of when a widget is hovered. Is there a way to modify this behaviour? I can't find anything online about editing the widget appearance when hovered.

Example of unhovered TextEdit widget:

![unhovered](https://user-images.githubusercontent.com/97441781/194344941-c82da7b4-55ba-45a6-a69b-dc425624d2b2.PNG)

Example of hovered TextEdit widget (notice the whitish outline of the widget):

![hovered](https://user-images.githubusercontent.com/97441781/194345606-abf15cdb-6a08-435a-b908-306596cb3636.PNG)

**Comment by paul-hansen** [comment, marked as answer]
You could probably to set the hover style to the same as the normal style.

If you only want to disable it for one element, something like this could work:
``` rust
let mut text = String::new();
let hovered_backup = ui.visuals().widgets.hovered;
ui.visuals_mut().widgets.hovered = ui.visuals().widgets.inactive;
// You can replace this with whatever you don't want to have hover effects 
ui.text_edit_multiline(&mut text);
// restore hover for other elements
ui.visuals_mut().widgets.hovered = hovered_backup;
```
